src/scripts/main.js

```diff
@@ -1,3 +1,86 @@
 'use strict';
 
-// write code here
+const itemOfRows = document.querySelector('tr');
+const table = document.querySelector('tbody');
+const rows = table.children;
+const rowAppenderButton = document.querySelector('.append-row');
+const rowRemoverButton = document.querySelector('.remove-row');
+const columnAppenderButton = document.querySelector('.append-column');
+const columnRemoverButton = document.querySelector('.remove-column');
+
+function toAddRow() {
+  rowAppenderButton.addEventListener('click', () => {
+    if (rows.length < 10) {
+      const copyRow = itemOfRows.cloneNode(true);
+
+      table.append(copyRow);
+    }
+
+    if (rows.length === 10) {
+      rowAppenderButton.setAttribute('disabled', 'true');
+    }
+
+    if (rows.length > 2) {
+      rowRemoverButton.removeAttribute('disabled');
+    }
+  });
+}
+
+function toRemoveRow() {
+  rowRemoverButton.addEventListener('click', () => {
+    if (rows.length > 2) {
+      rows[rows.length - 1].remove();
+    }
+
+    if (rows.length === 2) {
+      rowRemoverButton.setAttribute('disabled', 'true');
+    }
+
+    if (rows.length < 10) {
+      rowAppenderButton.removeAttribute('disabled');
+    }
```

Comment:
what if we make a separate function with this code in order not to repeat ourselves

Reply:
I hope I understood you)